### PR TITLE
 Add Feedback/Rate Us section done ! #290

### PR DESCRIPTION
**Pull request by Subhajit-2023-44**
Hello, PR
Add Feedback/Rate Us section with star rating widget - #290 
Done properly -

https://github.com/user-attachments/assets/24624dff-f436-4b54-8222-3cc82d6dca9b

Screenshot 2024-10-31 030545
Screenshot 2024-10-31 030554
Screenshot 2024-10-31 030611

## ✅ Checklist

Before submitting the PR, please make sure you have completed the following:

- [x] I have followed all the guidlines mentioned in CONTRIBUTING.md.
- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation (if applicable).
- [x] I have tested it locally and it works fine.
- [x] Any dependent changes have been merged and published in downstream modules.

## 🏷️ Types of Changes

What type of changes does your code introduce? (Check all that apply)

- [ ] Bug fix (non-breaking change which fixes an issue) 🐛
- [x] New feature (non-breaking change which adds functionality) ✨
- [ ] UI enhancement (non-breaking change which enhances UI) 🎨
- [ ] Documentation update 📚

### CODE REVIEW

**Comment by Subhajit-2023-44** [comment]
@Harshdev098 sir done, please merge my PR with level 3, I also add a star rating widget with feedback form..


Thank You

**Comment by Subhajit-2023-44** [comment]
@Harshdev098 sir

**Comment by Subhajit-2023-44** [comment]
@Harshdev098 sir

**Review by Harshdev098 — CHANGES_REQUESTED** [review]
I think you can make it more responsive

image

**Comment by Subhajit-2023-44** [comment]
> I think you can make it more responsive


What ? I didn't understand ... 

@Harshdev098 sir i added everything what i can do, Feedback form, toast message & also star rating widget...

Screenshot 2024-10-31 030554

If you check other projects, they give level 2/3 for feedback form and level 2 for star rating and another level 2 for toast message but I added everything in this PR because tomorrow was the last day, so...

@Harshdev098 sir please merge my PR 🙏🏻

File: public/css/style.css [hunk 1]
```diff
@@ -2241,3 +2242,174 @@ body {
...
+
+  background-color: #007bff;
+  color: white;
+  padding: 10px 15px;
+  border: none;
+  border-radius: 5px;
+  cursor: pointer;
+  margin-left: 183px;
+  margin-top: -33px; /* Add space above the button */
+  display: block; /* Ensure it takes the full width */
```
Comment (Harshdev098) [inline public/css/style.css:2306]: Make the display, `flex` and add some padding in x-axis of the `i` tag

**Review by Harshdev098 — CHANGES_REQUESTED** [review]

File: public/css/style.css (hunk 1 above)
Comment (Subhajit-2023-44) [reply public/css/style.css:2306]: > Make the display, `flex` and add some padding in x-axis of the `i` tag

😢okay wait i will give a new PR again

**Comment by Subhajit-2023-44** [comment]
@Harshdev098 sir done #375

**Pull request closed without merging**